### ISSUE TITLE
Change Command: <RESPAWN>

### ISSUE THREAD

**Issue by Bendzsike** [harbingerofme/DebugToolkit#138]
**Which Command**
`respawn`

**What should it do?**
Right now, if I use this command, I do respawn, as the command suggests, but I get the End Screen card, which can't be turned off, disabled, or anything, no matter what you do. You can hit Esc, you can hit anything, it will stay until you press the Continue button, and that will end the "run".

**Comment by SChinchi** [comment]
Just to be clear, this happens when you try to respawn after the last player alive has been killed. The reason for that is the game creates the EndGameReportPanel GameObject, whose UI interferes with the game.  Destroying that should do the job, but I have no idea how it would affect a multiplayer game.

The question is whether the command should do that to begin with.

**Comment by Bendzsike** [comment]
Yes, I figured it's because the last alive player died as well. Imo it would be perfectly fine if it either destroyed the GameObject together with respawning, or if there was a separate command for this.

**Comment by SChinchi** [comment]
That sounds much better. It can be a command that toggles it, like `god`.

**Comment by Bendzsike** [comment]
Yeah, I like that idea. Is trhere any estimate when will it get implemented, or if at all?

**Comment by SChinchi** [comment]
The functionality for this behavior is indirectly achieved with the `buddha` command. The idea being having a command that toggles preventGameOver would require it to be run before you die, just like buddha.

They have conceptual differences, such as not dying in general so your dios are not spent, or not being able to tell whether something would have killed you or legitimately left you at 1 hp. But practically if you intend to respawn after being killed, any dios are irrelevant.
